File: ShakesAndFidgetAPI/models/gearBase.js
var Sequelize = require('sequelize');

module.exports = function (sequelize) {
  const GearBase = sequelize.define('gearBase', {
    Id: {
      type: Sequelize.INTEGER,
      autoIncrement: true,
      primaryKey: true
    },
    Name: {
      type: Sequelize.STRING
    },
    ImageSource: {
      type: Sequelize.STRING
    },
    CharacterType: {
      type: Sequelize.STRING(4)
    },
    GearType: {
      type: Sequelize.STRING(7)
    },
    LevelMin: {
      type: Sequelize.INTEGER
    },
    Life: {
      type: Sequelize.INTEGER
    },
    Mana: {
      type: Sequelize.INTEGER
    },
    Energy: {
      type: Sequelize.INTEGER
    },
    Strength: {
      type: Sequelize.INTEGER
    },
    Agility: {
      type: Sequelize.INTEGER
    },
    Spirit: {
      type: Sequelize.INTEGER
    },
    Luck: {
      type: Sequelize.INTEGER
    },
    CriticalDamage: {
      type: Sequelize.INTEGER
    },
    MagicDamage: {
      type: Sequelize.INTEGER
    },
    PhysicalDamage: {
      type: Sequelize.INTEGER
    },
    CriticalProba: {
      type: Sequelize.INTEGER
    },
    PhysicalArmor: {
      type: Sequelize.INTEGER
    },
    MagicalArmor: {
      type: Sequelize.INTEGER
    }
  });

  return GearBase;
};

File: ShakesAndFidgetLibrary/Models/GearBase.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ShakesAndFidgetLibrary.Models
{
    public class GearBase : ModelBase
    {
        public String Name { get; set; }
        public String ImageSource { get; set; }
        public String CharacterType { get; set; }
        public String GearType { get; set; }
        public int LevelMin { get; set; }
        public int Life { get; set; }
        public int Mana { get; set; }
        public int Energy { get; set; }
        public int Strength { get; set; }
        public int Agility { get; set; }
        public int Spirit { get; set; }
        public int Luck { get; set; }
        public int CriticalDamage { get; set; }
        public int MagicDamage { get; set; }
        public int PhysicalDamage { get; set; }
        public int CriticalProba { get; set; }
        public int PhysicalArmor { get; set; }
        public int MagicalArmor { get; set; }
        public int Price { get; set; }

        public Gear ToGear()
        {
            Gear gear = new Gear
            {
                Name = Name,
                ImageSource = ImageSource,
                CharacterType = CharacterType,
                GearType = GearType,
                LevelMin = LevelMin,
                Life = Life,
                Mana = Mana,
                Energy = Energy,
                Strength = Strength,
                Agility = Agility,
                Spirit = Spirit,
                Luck = Luck,
                CriticalDamage = CriticalDamage,
                MagicDamage = MagicDamage,
                PhysicalDamage = PhysicalDamage,
                CriticalProba = CriticalProba,
                PhysicalArmor = PhysicalArmor,
                MagicalArmor = MagicalArmor,
                Price = Price
            };
            return gear;
        }

        public Stats ToStats()
        {
            Stats stats = new Stats
            {
                Life = Life,
                Mana = Mana,
                Energy = Energy,
                Strength = Strength,
                Agility = Agility,
                Spirit = Spirit,
                Luck = Luck,
                CriticalDamage = CriticalDamage,
                MagicDamage = MagicDamage,
                PhysicalDamage = PhysicalDamage,
                CriticalProba = CriticalProba,
                PhysicalArmor = PhysicalArmor,
                MagicalArmor = MagicalArmor
            };
            return stats;
        }
    }
}
